src/RTC_RV3032C7.cpp

```diff
@@ -0,0 +1,317 @@
+#include "RTClib.h"
+// TODO: only what is needed in basic examples is implemented right now.
+//       The RV3032C7 chip supports a lot more functionality...
+
+//#define DEBUG_SERIAL Serial
+//#define DEBUG_SERIAL SerialUSB
+//#ifdef DEBUG_SERIAL
+//     #include <Arduino.h>
+//#endif
```

Comment:
This should probably be removed.

Reply:
Agreed! It will be removed in the next update, coming soon.